internal/pkg/skuba/addons/addons.go

```diff
@@ -275,6 +276,15 @@ func (addon Addon) Apply(client clientset.Interface, addonConfiguration AddonCon
 		klog.Errorf("failed to run kubectl apply: %s", combinedOutput)
 		return err
 	}
+
+	replicaHelper, err := replica.NewHelper(client)
+	if err != nil {
+		return err
+	}
+	if err := replicaHelper.UpdateNodes(); err != nil {
```

Comment:
`replicaHelper.UpdateNodes()` will impact all deployments with the label `caasp.suse.com/skuba-replica-ha` that means each addon application will repeat updating all deployments even some of them already update with correct podAntiAffinity.

I would suggest to move this operation to upper level to make sure only one call after all addons deployment/upgrade at http://github.com/innobead/skuba/blob/master/internal/pkg/skuba/addons/addons.go#L136-L136 to avoid needless calls.

Reply:
Not sure I understand this one correctly. Not sure by moving `replicaHelper.UpdateNodes()` to upper level. 

Currently it will loop through all addon and update its associate deployment because replica size might differ for every addon.

Reply:
Aaah, ok I get what you mean.

Reply:
Moved to upper level after `DeployAddons`